ibis/backends/tests/test_generic.py

```diff
@@ -489,7 +497,19 @@ def test_dropna_invalid(alltypes):
 
 @pytest.mark.parametrize("how", ["any", "all"])
 @pytest.mark.parametrize(
-    "subset", [None, [], "col_1", ["col_1", "col_2"], ["col_1", "col_3"]]
+    "subset",
+    [
+        None,
+        # The combination with any works but not the one with all
+        # Question: Is there a way to specifically suppress subset1-all?
```

Comment:
Unfortunately not.  You'll need to unroll the nested parametrization to xfail the specific combination.

Reply:
Thanks, for the answer and also good catch (a leftover comment I've missed :sweat_smile:). I will remove it.